Database/Corrections/cataQuestFixes.lua

```diff
@@ -1156,6 +1160,9 @@ function CataQuestFixes.Load()
         [13646] = { -- Astranaar Bound
             [questKeys.preQuestSingle] = {26464},
         },
+        [13653] = { -- Crisis at Splintertree
+            [questKeys.extraObjectives] = {{nil,Questie.ICON_TYPE_TALK,l10n("Speak to Gorka"),0,{{"monster",33421}}}},
```

Comment:
Same as above with `l10n(...)`. On top it kind of looks like the `objectives` of this quest is a bunny NPC. I'd say that the objectives can be cleared and instead a `triggerEnd` can be added with the location of where the quest will be complete. That way users can at least see where the quests will be done.

Reply:
https://github.com/Questie/Questie/pull/6359/commits/aabb11e3716d0fa09366997b071585d49b673920

@Cabro I asked you about this on discord a few days ago - would you mind checking I've done correctly please?

Reply:
Looks good